common/bloodlines/wc_bloodlines.txt

```diff
@@ -350,6 +350,32 @@ lireesa = { # 603 only, no related event for earlier start
 	flags = { historical_bloodline bloodline_hunter_training }
```

Comment:
Don't forget to uncomment `d_throsic`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/80977757-39f5a700-8e36-11ea-99c9-d78c2c9e008f.png)

</details>

Reply:
d_throsic doesn't exist in history, also shouldn't have a holder at gamestart